commit 76a3ccabf25bc3a52c3377bfdfc3ecc16499180b
Author: Henry Ptasinski <henryp@broadcom.com>
Date:   Sun Sep 26 13:18:38 2010 -0700

    staging: brcm80211: Fix compile issue when BRCM80211_PCI is not set.
    
    Signed-off-by: Henry Ptasinski <henryp@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/Makefile
+++ b/drivers/staging/brcm80211/Makefile
@@ -67,10 +67,9 @@ PCIFILES := \
 
 MODULEPFX := brcm80211
 
-obj-m	+= $(MODULEPFX).o
-
 # PCI driver
 ifeq ($(CONFIG_BRCM80211_PCI),y)
+obj-m	+= $(MODULEPFX).o
 ccflags-y += $(PCI_CFLAGS)
 $(MODULEPFX)-objs	= $(BRCM80211_OFILES) $(PCIFILES)
 endif
